keyboards/handwired/dactyl_manuform/5x6_68/keymaps/default/keymap.c

```diff
@@ -5,39 +5,45 @@
-                 KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   ,                          KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_BSLS,
-                 KC_ESC , SFT_A  , CTL_S  , ALT_D  , GUI_F  , KC_G   ,                          KC_H   , GUI_J  , ALT_K  , CTL_L  , SFT_SC , KC_QUOT,
-                 KC_LSFT, KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   ,                          KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,
-                 KC_GRV , KC_INS , KC_LEFT, KC_RGHT,                                                              KC_DOWN, KC_UP  , KC_LBRC, KC_RBRC,
-                                                     KC_LCTL, KC_LALT, KC_HOME,        KC_DEL , KC_PGUP, KC_BSPC,
-                                                     KC_SPC , KC_DEL , KC_END ,        MO(_LAYER2), KC_PGDN, KC_ENT
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,           KC_6,    KC_7,  KC_8,    KC_9,   KC_0,    KC_GRV,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,           KC_Y,    KC_U,  KC_I,    KC_O,   KC_P,    KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,           KC_H,    KC_J,  KC_K,    KC_L,   KC_SCLN, KC_QUOT,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,           KC_N,    KC_M,  KC_COMM, KC_DOT, KC_SLSH, KC_RSFT,
+        KC_LCTL, KC_LALT, KC_LBRC, KC_RBRC,                                          KC_MINS, KC_EQL, KC_RALT, KC_RCTL,
+                                            RAISE,   KC_SPC,         KC_ENT,  LOWER,
+                                            KC_HOME, KC_BSPC,        KC_DEL,  KC_END,
+                                            KC_PGUP, KC_LGUI,        KC_RGUI, KC_PGDN
     ),
-    [_LAYER2] = LAYOUT(
-                 XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, QK_BOOT,
-                 XXXXXXX, IMPORTS, XXXXXXX, USAGES , RENAME , XXXXXXX,                          XXXXXXX, KC_VOLD, KC_MUTE, KC_VOLU, XXXXXXX, XXXXXXX,
-                 XXXXXXX, LINES  , XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                          XXXXXXX, KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX,
-                 XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
-                 XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                                                              XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
-                                                     XXXXXXX, XXXXXXX, XXXXXXX,         XXXXXXX,XXXXXXX, XXXXXXX, 
-                                                     XXXXXXX, XXXXXXX, XXXXXXX,         _______,XXXXXXX, XXXXXXX
+
+    [_LOWER] = LAYOUT(
+        KC_TILD,KC_EXLM, KC_AT ,KC_HASH,KC_DLR ,KC_PERC,        KC_CIRC,KC_AMPR,KC_ASTR,KC_LPRN,KC_RPRN,_______,
```

Comment:
would prefer spaces between the keycodes, as it makes it easier to read.

Reply:
> would prefer spaces between the keycodes, as it makes it easier to read.

fair point 🙂